### PR TITLE
[FIX] Invalid version

### PR DESCRIPTION
**Pull request by djorkaeffalexandre**
@RocketChat/reactnative 

After we start `serverSelectRequest` the `RocketChat.connect` is started and call `login`, when login success we call `fetchCustomEmojis`, `fetchPermissions` and this access `redux.getState().server.version`, but if we get this before `serverSelectSuccess` we have a null version.. it can occur when `/api/info` is slower than `/login`..

Other possible approach is create a action `setVersion` and call it when `getServerInfo` 🤔 ...
What do you think?

### CODE REVIEW

File: app/lib/methods/getCustomEmojis.js [hunk 1]
```diff
@@ -84,14 +84,18 @@ export async function setCustomEmojis() {
 export function getCustomEmojis() {
 	return new Promise(async(resolve) => {
 		try {
-			const serverVersion = reduxStore.getState().server.version;
+			const { server } = reduxStore.getState().server;
+			const serversDB = database.servers;
+			const serversCollection = serversDB.collections.get('servers');
+			const serverInfo = await serversCollection.find(server);
+
```
Comment (rodrigok) [inline app/lib/methods/getCustomEmojis.js]: Is it possible to create a helper to prevent duplicate this code everywhere we need to check the version?

Comment (djorkaeffalexandre) [reply app/lib/methods/getCustomEmojis.js]: Yeah! 👍

Comment (djorkaeffalexandre) [reply app/lib/methods/getCustomEmojis.js]: @rodrigok what you think about it?
> Other possible approach is create a action setVersion and call it when getApiInfo 🤔 ...

Comment (rodrigok) [reply app/lib/methods/getCustomEmojis.js]: @djorkaeffalexandre I'm not sure how those actions work so can't opine on that matter :(

Comment (djorkaeffalexandre) [reply app/lib/methods/getCustomEmojis.js]: Ok! Thx :)

**Comment by djorkaeffalexandre** [comment]
I found another way to reproduce it and will work on it..
Steps:
* Login on open.rocket.chat
* Add server
* Login on stable.rocket.chat
* Go to open.rocket.chat
* Add server
* Write stable.rocket.chat
* It raises a `version = null` and the app will be connecting infinitely.

**Pull request merged**